task/06-conditions-n-loops-tasks.js

```diff
@@ -209,7 +263,12 @@ function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
  * 'noon' => 'noon'
  */
 function reverseString(str) {
-    throw new Error('Not implemented');
+    let resultReverseString = '';
```

Comment:
For this task, you can also use a simpler solution through the "reverse" method

Reply:
Fixed.